<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>渐变</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QPushButton" name="conicalGradientButton">
     <property name="text">
      <string>圆锥渐变</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="linearGradientButton">
     <property name="text">
      <string>线性渐变</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="radialGradientButton">
     <property name="text">
      <string>径向渐变</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="linearGradientAlgorithmButton">
     <property name="text">
      <string>线性渐变算法</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="linearGradientAlgorithm2Button">
     <property name="text">
      <string>线性渐变算法-任意角度</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
